examples/webgl_postprocessing_glitch.html

```diff
@@ -6,8 +6,22 @@
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link type="text/css" rel="stylesheet" href="main.css">
 	</head>
+	<style>
+		#overlay {
+			flex-direction: column;
+		}
+	</style>
 	<body>
 
+		<div id="overlay">
+			<h3>Photosensitive Epilepsy Warning</h3>
+			<p class="warning">
+				This demo contains strong and frequent flashes which may trigger a potential <strong>Photosensitive Epilepsy.</strong>
+				<br/>
+				If you got any uncomfortable, press <strong>Ctrl + W</strong> or <strong>Command + W</strong> to close this page.
```

Comment:
I don't think we have to explain user how to close a browser tab 😉 .

Reply:
I see... Let me commit another change